### ISSUE TITLE
Test the chart with Cypress

### ISSUE THREAD

**Issue by dbpiper** [dbpiper/Euclid#42]
As a developer I would like to have verified tests proving that the chart component works so that I can be sure my future changes introduce no regressions and that things work as expected.

In Sprint 6 we fixed #25 which was essentially a [Spike issue](https://www.scaledagileframework.com/spikes/) to determine how we want to test React components, and also to actually do some testing using whatever method was chosen. We are using Cypress for this as it seems to be the best tool for the job, and allows testing of React components just as the user will use them, rather than some artificial test which simply sets props or something.

However, we only tested the search drop-down boxes and not the chart component. So this should be done in this issue. I don't expect that we will be able to reliably test the displaying of the chart data, since this would be quite complex and flaky due to relying on actual data for the test, we would have to somehow mock the data as in the server's unit tests. However, we shouldn't have to be cause this is simply using Recharts and so we should just assume that their code works (or file issues if it doesn't). In addition, we are already unit testing the data going  into the server and the data received from the server on the client's business logic, so we can be pretty confident that this testing verifies that our chart will be drawn correctly.

We should focus on testing that the chart works in every way **except** for the actual drawing of data. Such as having the correct labels, tooltip, etc.

**Comment by dbpiper** [comment]
Upon reflection it is actually not going to be feasible to test this component without some sort of mock data. I think the testing that is done should just be done manually, we are at least guaranteed to have a connection to the DB because of the header tests. However, testing any part of this component is inherently dependent on the data in the DB which would make any testing attempt very flaky. We could isolate the component and test it in isolation with something like Storybook, and this is probably what we should do. 

However, this is not something that we can do at this time as we don't yet have Storybook setup for Euclid and this would require its own separate issue.

**Comment by dbpiper** [comment]
Taking this out of Sprint 7 due to being blocked by Storybook adoption.

**Comment by dbpiper** [comment]
#50 is a duplicate of this. However, since #50 is more specific about how to handle the problem, I will close this one and keep #50 open instead to track this.
